src/site/content/en/blog/building-a-tabs-component/index.md

```diff
@@ -22,9 +22,14 @@ that is responsive, supports multiple device inputs, and works across browsers.
 Try the [demo](https://gui-challenges.web.app/tabs/dist/).
 
 <figure class="w-figure w-figure--fullbleed">
-  <video playsinline controls autoplay loop muted class="w-screenshot">
-    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/gui-tabs/mouse-and-keyboard.mp4">
-  </video>
+  {% Video 
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/IBDNCMVCysfM9fYC9bnP.mp4",
+    className="w-screenshot", 
+    playsinline="true", 
```

Comment:
```suggestion
```

It looks like if you set autoplay then we go ahead and set playsinline for you.
https://github.com/GoogleChrome/web.dev/blob/main/src/site/_includes/components/Video.js#L45